yjit.c

```diff
@@ -120,6 +120,55 @@ rb_yarv_insn_len(VALUE insn)
     return insn_len(insn);
 }
 
+unsigned int
+rb_yarv_vm_ci_argc(const struct rb_callinfo *ci) {
```

Comment:
I think just `rb_` would be good here. We just need to add the `rb_` prefix because this will export a symbol, but otherwise we should try to keep the names short.

Reply:
Sounds good. I'll go through and do an s/rb_yarv_/rb_/g everywhere.